docs/topics/Contribution-Guide.md

```diff
@@ -0,0 +1,34 @@
+# Contribution Guide
+
+Those who wish to make open-source contributions to the API must adhere to this contribution guide.
+
+## Claiming an Issue
+
+First-time contributors may volunteer to take low-priority tasks to work on. We request first-time contributors take low-priority issues due to the length of time it takes a first-time developer to contribute to the project.
```

Comment:
This seems a bit wordy, is the reasoning here about difficulty of onboarding, or time needed for code review, or something else?